ios/DataModels/PrimerSettingsRN.swift

```diff
@@ -1,168 +1,13 @@
-                firstName: customer?.firstName,
-                lastName: customer?.lastName,
-                emailAddress: customer?.email,
-                homePhoneNumber: nil,
-                mobilePhoneNumber: customer?.phone,
-                workPhoneNumber: nil,
-                billingAddress: address
-            )
-        )
-    }
-}
-
-fileprivate struct OrderRN: Decodable {
-    let id: String?
-    let amount: Int?
-    let currency: Currency?
-    // FIXME: Move countryCode only within Address (RN & Android)
-    let countryCode: CountryCode?
-    let items: [OrderItemRN]?
-    let shipping: AddressRN?
-}
-
-fileprivate extension OrderRN {
-    var itemsFormatted: [OrderItem]? {
-        return items?.compactMap { try? $0.primerFormat() }
-    }
-}
-
-//extension Array where Element == OrderItemRN {
-//    var primerFormat: [OrderItem] {
-//        return self.map { $0.primerFormat }
-//    }
-//}
-
-fileprivate struct OrderItemRN: Decodable {
-    let name: String
-    let unitAmount: Int?
-    let quantity: Int
-    let isPending: Bool?
-}
-
-fileprivate extension OrderItemRN {
-    func primerFormat() throws -> OrderItem {
-        return try OrderItem(
-            name: name,
-            unitAmount: unitAmount,
-            quantity: quantity,
-            isPending: isPending ?? false
-        )
-    }
-}
-
-fileprivate struct BusinessRN: Decodable {
-    let name: String
-    let registrationNumber: String?
-    let email: String?
-    let phone: String?
-    let address: AddressRN?
-}
-
-fileprivate extension BusinessRN {
-    var primerFormat: BusinessDetails? {
-        guard let address = address else {
-            return nil
-        }
-        
-        return BusinessDetails(
-            name: name,
-            address: address.primerFormat
-        )
-    }
-}
-
-fileprivate struct CustomerRN: Decodable {
-    let id: String?
-    let firstName: String?
-    let lastName: String?
-    let email: String?
-    let phone: String?
-    let billing: AddressRN?
-}
-
-fileprivate struct AddressRN: Decodable {
-    let line1: String?
-    let line2: String?
-    let postalCode: String?
-    let city: String?
-    let state: String?
-    let country: CountryCode?
-}
-
-fileprivate extension AddressRN {
-    var primerFormat: Address {
-        return Address(
-            addressLine1: line1,
-            addressLine2: line2,
-            city: city,
-            state: state,
-            countryCode: country?.rawValue,
-            postalCode: postalCode
-        )
-    }
-}
-
-fileprivate struct OptionsRN: Decodable {
-    let isResultScreenEnabled: Bool?
-    let isLoadingScreenEnabled: Bool?
-    let isFullScreenEnabled: Bool?
-    let is3DSOnVaultingEnabled: Bool?
-    let is3DSDevelopmentModeEnabled: Bool?
-    let locale: String?
-    let ios: IosOptionsRN?
-}
-
-fileprivate extension String {
-    var localeFormatted: Locale {
-        return Locale(identifier: self)
+        return PrimerSettings()
```

Comment:
can we double-check whether the whole section can be removed 🙏

Reply:
done